### PR TITLE
Colorjitter

### PR DESCRIPTION
**Pull request by priba**
Add Color Jitter to Data augmentation

### CODE REVIEW

File: kornia/augmentation/augmentations.py [hunk 1]
```diff
@@ -73,6 +76,52 @@ def forward(self, input: UnionType) -> UnionType:  # type: ignore
         return random_hflip(input, p=self.p, return_transform=self.return_transform)
 
 
+class ColorJitter(nn.Module):
+
+    r"""Change the brightness, contrast, saturation and  hue randomly given tensor image or a batch of tensor images.
```
Comment (ananyamukh6) [inline kornia/augmentation/augmentations.py]:
```suggestion
    r"""Change the brightness, contrast, saturation and hue randomly given tensor image or a batch of tensor images.
```

File: examples/data_augmentation.py [hunk 2]
```diff
@@ -12,6 +12,9 @@
 import torch.nn as nn
 import torch.optim as optim
 from torch.utils.data import Dataset, DataLoader
+import torchvision
+from matplotlib import pyplot as plt
+import cv2
```
Comment (edgarriba) [inline examples/data_augmentation.py]: have you checked that the examples are well generated ?

File: kornia/augmentation/augmentations.py [hunk 3]
```diff
@@ -124,3 +174,102 @@ def random_hflip(input: torch.Tensor, p: float = 0.5, return_transform: bool = F
...
+    hue_bound = _input_check(hue, 'hue', bounds=(-.5, .5))
+
+    device: torch.device = input.device
+    dtype: torch.dtype = input.dtype
+
+    input = input.unsqueeze(0)
+    input = input.view((-1, (*input.shape[-3:])))
+
+    brightness_distribution = Uniform(brightness_bound[0], brightness_bound[1])
+    brightness_factor = brightness_distribution.rsample([input.shape[0]])
```
Comment (edgarriba) [inline kornia/augmentation/augmentations.py]: in order to make the gpu integration tests easy -> `factor_bound.to(input.device).to(input.dtype)`

same for the rest

File: test/augmentation/test_augmentation.py [hunk 4]
```diff
@@ -142,3 +142,485 @@ def test_gradcheck(self):
...
+        f = ColorJitter()
+        f1 = ColorJitter(return_transform=True)
+
+        input = torch.rand(3, 5, 5)  # 3 x 5 x 5
+
+        expected = input
+
+        expected_transform = torch.eye(3).unsqueeze(0)  # 3 x 3
+
+        assert_allclose(f(input), expected, atol=1e-4, rtol=1e-5)
```
Comment (edgarriba) [inline test/augmentation/test_augmentation.py]: is it noisy ?

Comment (priba) [reply test/augmentation/test_augmentation.py]: It makes use of the HSV space. It is noisy as you can check here: https://github.com/kornia/kornia/blob/master/test/color/test_hsv.py#L35

File: kornia/augmentation/augmentations.py (hunk 3 above)
Comment (priba) [reply kornia/augmentation/augmentations.py]: Corrected in the following commit

File: examples/data_augmentation.py (hunk 2 above)
Comment (priba) [reply examples/data_augmentation.py]: I recovered the previous file and with the help of @dvd42 we will generate an example for Data augmentation with all the modules

**Pull request merged**